css-two-column-layout/styles.css

```diff
@@ -0,0 +1,77 @@
+body {
+  width: 80%;
+  margin: auto;
+}
+
+div {
+  justify-content: center;
+}
+
+.heading {
+  display: flex;
```

Comment:
If you want to use `display: flex` on an element, that's usually an indicator it should be a `row` in your layout. You can apply the additional styling here as a separate class in addition to `row`, but if you are trying to build a layout using proper "layout classes" `row` is your go-to tool for making a flexbox